src/soilbiogeochem/SoilBiogeochemDecompCascadeBGCMod.F90

```diff
@@ -525,11 +440,28 @@ subroutine init_decompcascade_bgc(bounds, soilbiogeochem_state_inst, soilstate_i
       is_soil(i_soil3) = .true.
       is_cwd(i_soil3) = .false.
       initial_cn_ratio(i_soil3) = cn_s3
-      initial_stock(i_soil3) = params_inst%initial_Cstocks(3)
+      initial_stock(i_soil3) = params_inst%initial_Cstocks(i_soil3)
       is_metabolic(i_soil3) = .false.
       is_cellulose(i_soil3) = .false.
       is_lignin(i_soil3) = .false.
 
+      if (.not. use_fates) then
+         ! CWD
```

Comment:
We discussed moving this CWD if-block back between lignin (currently i_litr3) and i_soil1 a few lines up from here where it used to be. HOWEVER, I just reminded myself that I moved this if-block to the end of the pools because I moved `initial_Cstocks_bgc` from the namelist to the params file:
`initial_Cstocks_bgc = 0, 0, 0, 200, 200, 200, 0, _ ;`
The order of these values matters and I placed the CWD value last because it is optional depending on whether we are running with FATES or not.

Reply:
Ahh, that is important to be able to flip between FATES on and off.